src/index.php

```diff
@@ -214,6 +214,8 @@
 $app->get('/app/{appName}/{projectId}',     'Site\Controller\App::view');
 $app->get('/app/{appName}/',    'Site\Controller\App::view')->value('appName', 'projects');
 $app->get('/app/{appName}',     'Site\Controller\App::view')->value('appName', 'projects');
+$app->get('/redirect/{appName}/',    'Site\Controller\Redirect::view')->value('appName', 'project');
+$app->get('/redirect/{appName}',     'Site\Controller\Redirect::view')->value('appName', 'project');
```

Comment:
Can we get this route/controller working like this?

```suggestion
$app->get('/redirect/project/',    'Site\Controller\Redirect::view');
$app->get('/redirect/project',     'Site\Controller\Redirect::view');
```

Reply:
Done.